From 10fdc8d749f70cc573e407f11e5f850ee19a70d6 Mon Sep 17 00:00:00 2001
From: Elmar Pruesse <elmar.pruesse@ucdenver.edu>
Date: Wed, 15 May 2019 19:12:03 -0600
Subject: [PATCH 4/4] Disable CMake c++14 pinning

We need to build c++17 because BOOST is not ABI compatible across
standard versions. Unlike Autotools, CMake does not give any useful
means for overriding the standard from outside. :(
---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1720c10..3e7f059 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,7 +7,6 @@ include(${CMAKE_ROOT}/Modules/ExternalProject.cmake)
 find_library(ZLIB z)
 find_library(HTSLIB hts)
 
-set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
-- 
2.14.0

